[CI] The deprecation list is showing weird things

The deprecation list is showing weird things on https://pointcloudlibrary.github.io/documentation/deprecated.html but not on http://docs.pointclouds.org/trunk/deprecated.html.

EDIT: and CI is reporting this
```
/__w/1/s/common/include/pcl/common/io.h:82: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
/__w/1/s/common/include/pcl/common/io.h:111: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
/__w/1/s/common/include/pcl/common/io.h:121: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
/__w/1/s/common/include/pcl/common/io.h:335: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
/__w/1/s/common/include/pcl/point_cloud.h:142: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
/__w/1/s/segmentation/include/pcl/segmentation/supervoxel_clustering.h:291: warning: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)
```
and more new ones

_Originally posted by @aPonza in https://github.com/PointCloudLibrary/pcl/pull/3808#issuecomment-605535873_

--- a/.ci/azure-pipelines/documentation.yaml
+++ b/.ci/azure-pipelines/documentation.yaml
@@ -33,6 +33,10 @@ jobs:
           cmake --build . -- doc tutorials advanced
         displayName: 'Build Documentation'
       - script: |
+          cd $BUILD_DIR
+          sed -r -e 's/([0-9]+)\s\.\s([0-9]+)\s/\1.\2/' doc/doxygen/html/deprecated.html
+        displayName: 'Remove extra spaces in Doxygen''s Deprecated List'
+      - script: |
           git config --global user.email "documentation@pointclouds.org"
           git config --global user.name "PointCloudLibrary (via Azure Pipelines)"
           echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config

--- a/doc/doxygen/doxyfile.in
+++ b/doc/doxygen/doxyfile.in
@@ -312,7 +312,7 @@ PREDEFINED =           = "HAVE_QHULL=1" \
                          "HAVE_RSSDK=1" \
                          "DOXYGEN_ONLY=1" \
                          "_WIN32=1" \
-                         "PCL_DEPRECATED(1, 12, x)=/** \deprecated x */"
+                         "PCL_DEPRECATED(major,minor,message)=/** \deprecated Scheduled for removal in version major.\ minor: message */"
 EXPAND_AS_DEFINED      =
 SKIP_FUNCTION_MACROS   = YES
 

